arho_feature_template/project/layers/plan_layers.py

```diff
@@ -117,6 +155,42 @@ class RegulationGroupAssociationLayer(AbstractPlanLayer):
         )
     )
 
+    layer_name_to_attribute_map: ClassVar[dict[str, str]] = {
+        LandUsePointLayer.name: "land_use_point_id",
+        OtherAreaLayer.name: "other_area_id",
+        OtherPointLayer.name: "other_point_id",
+        LandUseAreaLayer.name: "land_use_area_id",
+        LineLayer.name: "line_id",
+    }
+
+    @classmethod
+    def feature_from(
+        cls,
+        regulation_group_id: str | None,
+        layer_name: str | None = None,
+        feature_id: str | None = None,
+        plan_id: str | None = None,
+    ) -> QgsFeature:
```

Comment:
Käsittele tässä Kaava ja sen yleismääräysryhmä samalla tavalla kuin kaavamääräysryhmät, eli jos layer_name on Plan.name, niin tallenna feature_id plan_id -kenttään.
Kaikki parametrit on tällöin pakollisia eikä voi olla None.

```suggestion
    def feature_from(
        cls,
        regulation_group_id: str,
        layer_name: str,
        feature_id: str,
    ) -> QgsFeature:
```